docs/tutorials/gettingstartedexample/gse-create-tdms.md

```diff
@@ -0,0 +1,193 @@
+    
+9. Run the TDMS File Viewer VI and verify that all of the RAW12 Long Packets are surrounded by Line Start and Line End packets. The display should look like the image below.
+    
+    ![Viewer Line Sync Packets](../../images/PXIe-148X-TDMSFileViewer-LineSyncPackets.png)
+    
+
+## Evenly spacing long packets to fill the entire time window for a frame    
+This tutorial is going to show you how to adjust the interpacket delay to spread the line packets out evenly throughout the frame period. You will use the TDMS file Viewer to look at the packets and see how you can adjust the timing to achieve a 30 FPS TDMS file.
+    
+> Note: For the purposes of this tutorial, all input control values not specified should be left as the default value.
+    
+1. Double click the Create CSI-2 Packet TDMS Files VI in the LabVIEW project.
+
+2. Set the **Number of Frames** control to 2.
+    
+3. Set the **desired frame rate (fps)** to 30.
+    
+4. Run the VI and look at the **Actual Frame Rate (fps)** indicator and note the FPS displayed is 30.
+    
+5. Double click the TDMS File Viewer VI in the LabVIEW project.
+    
+6. Select the correct location for the **TDMS File Directory** control. By default, TDMS files generated were saved to <font face = "courier new">\<yourprojectdir\>\\Host\\Gen\\TDMS Files</font>.
+    
+7. Run the TDMS File Viewer VI and look at the timestamps of the first Frame End and second Frame Start packets on packet indexes 1081 and 1082. Note that there is a remaining frame period of 20 milliseconds. We can adjust our interpacket delay so that there is more time between each line packet and still achieve 30 FPS. Note the image below.
+    
+    ![Large Frame Period](../../images/PXIe-148X-TDMSFileViewer-LargeFramePeriod.png)
+    
+8. Set the **line blanking (cycles)** to 4650 and regenerate the TDMS file. You should see that **Actual Frame Rate (fps)** is still 30 FPS.
+    
+9. Run the TDMS File Viewer VI and look at the Frame End and Frame Start packets on packet indexes 1081 and 1082. Now there should be a very small remaining frame period. We have successfully moved that delay in between each of the packets. Data sets with higher interpacket delays are easier for the Generation Example to run without packet timing errors.
+    
+## Create CSI-2 Packet TDMS Files VI Help
+    
+![Open Create CSI-2 Packet TDMS Files Front Panel](../../images/PXIe-148X-CreateTDMS-FrontPanel.png)
+    
+    
+### General
+- **Serial Input Channels** - Array of strings representing the channels you will generate TDMS files for. Generated TDMS files have the prefix “SIx_” to indicate it is associated with a channel ‘SOx’. Although the prefix contains ‘SI’ suggesting ‘serial input’, when used with the Generation GSE, the TDMS file actually associates a specific serial output channel e.g. ‘SO0’.
```

Comment:
Update first sentence, per Jon the grammar king:

Array of strings representing the channels for which you will generate TDMS files.